packages/sdk/typescript/human-protocol-sdk/src/escrow.ts

```diff
@@ -937,6 +937,7 @@ export class EscrowClient extends BaseEthersClient {
     amounts: bigint[],
     finalResultsUrl: string,
     finalResultsHash: string,
+    forceComplete = false,
```

Comment:
We need to update python SDK in the same way

Reply:
Fixed at https://github.com/humanprotocol/human-protocol/pull/3012/commits/867d0cf63bbf77e8989fa55508f16b99480cb089